README.md

```diff
@@ -1,41 +1,87 @@
+- [ ] Update [dependabot.yml](.github/dependabot.yml) with the [configuration for your project](https://docs.github.com/en/code-security/supply-chain-security/keeping-your-dependencies-updated-automatically/configuration-options-for-dependency-updates).
+- [ ] Replace `https://github.com/ably-labs/ably-labs-template-repo/issues` with the actual link of the repo in the [CONTRIBUTING.md](CONTRIBUTING.md) file.
+- [ ] Update this README so it provides enough information for people to understand how it works, how to run it locally and how it can be deployed to the cloud (see [GitHub](https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-readmes)).
+- [ ] Add a GitHub workflow to build/test/deploy your application. Use the [Ably Control API GitHub action](https://github.com/ably-labs/ably-control-api-action) to avoid creating Ably apps/API keys manually.
+- [ ] Add this repository to the [selected repositories in the Ably Labs org](https://github.com/organizations/ably-labs/settings/actions) that are allowed to run GitHub Actions.
 
 Once you're done, remove this section from the README. Good luck! 💪
 
----
 
-![Place eye candy header image here](https://placekitten.com/640/360)
 
-*// Place eye candy header image here ⬆️*
+# Netlify Identity & JWT authentication
+
+
+![Netlify + JWT + Ably](./assets/netlify-ably-jwt.png)
 
 ## Description
 
 // Explanation of the contents of the repository. Describe the use case.
 
-## Tech stack
 
-![A high level architecture diagram tells more than a 1000 words](https://placekitten.com/480/240)
 
-*// A high level architecture diagram tells more than a 1000 words ⬆️*
 
-The project uses the following components:
+## What you'll need
+
+1. An Ably API Key, sign up for a [free account](https://ably.com/sign-up)
+1. A [Netlify](https://netlify.com) account to host and manage your registered users.
+1. A [Github](https://github.com) account is required to link Netlify to the repository.
+
+# Overview of architecture
+
+This is the overview of our app. Our user experience requires a new user to register and confirm their email address to activate themselves with our app before they can log-in. When they login we validate them with Netlify Identity and check that they have not been flagged as Banned. Bad actors are not issued with a JWT token. Valid users are issued a token, to authenticate with Ably, and the authentication URL carries the user's unique ID.
+
+![](./assets/user-experience-netily-functions.png)
+
+The Netlify Identity allows us to administer users by editing metadata associated with their account, flagging a bad actor is a matter of assigning them a role via the Netlify dashboard.
+
+## Initial steps
+1. Fork this repository
+1. Use Netlify dashboard to create a new project, and select your fork as the source.
+1. Add the environment variables to the new Netlify app from the site settings section, under `Build & deploy`
```

Comment:
It would be useful to give examples of these values, from a users perspective, it's unclear what I should set for `ABLY_CAPABILITY` for example

Reply:
Yes, the blog has a better explanation and lnks to capabilities.
I will update the README with the ENV settings and better examplesof the capability. 

Do you think `{"*":["*"]}` is a sensible default? or should we recommend a default restrictions?

Reply:
I think recommend a capability with a narrow scope (e.g. just subscribe on a single channel like `{"my-channel": ["subscribe"]}`), and then link to the capability docs.